app.rb

```diff
@@ -0,0 +1,100 @@
+require_relative 'book'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'person'
+require_relative 'rental'
+
+def option1
+  puts '** Books **'
+  ObjectSpace.each_object(Book) do |obj|
+    puts "Title: #{obj.title} Author: #{obj.author}"
+  end
+end
+
+def option2
+  puts '** Teachers **'
+  ObjectSpace.each_object(Teacher) do |obj|
+    puts "Id: #{obj.id} Name: #{obj.name} Age: #{obj.age} Specialization: #{obj.specialization}"
+  end
+  puts '** Students **'
+  ObjectSpace.each_object(Student) do |obj|
+    puts "Id: #{obj.id} Name: #{obj.name} Age: #{obj.age}"
+  end
+end
+
+def opcion3
+  puts 'Do you want to create a student (1) or a teacher (2) [Input the number]:'
+  target = gets.chomp
+  option_student if target == '1'
+  option_teacher if target == '2'
+end
```

Comment:
- [ ]  Great job so far @haftamudesta Your App works great. There are just a few changes you need to make and you'll be on your way to the next milestone. :+1: 
- [ ] While testing out your UI, I noticed that when I try to create a student, I am unable to complete the process and it ends up throwing an error and breaking the app. However, from the example video, you can see that a user should be able to create both a student and a teacher without any errors or the app breaking. So kindly fix that :wink: 

- Video Reference

[Screencast from 07-01-2023 01:03:44 PM.webm](https://github.com/haftamudesta/school-library/assets/102699958/00b87cf5-d727-4de0-8f03-4665d26e4282)